packages/vue/src/components/molecules/SfModal/SfModal.html

```diff
@@ -7,7 +7,7 @@
     @click="checkPersistence">
   </SfOverlay>
   <transition :name="transitionModal">
-    <div class="sf-modal__container" v-show="visible">
+    <div class="sf-modal__container" v-model="visible" v-show="visible">
```

Comment:
@aniamusial v-model on div? why?

Reply:
Ops, sorry @aniamusial maybe I have expressed myself badly about it, `v-model` is actually for `<SfModal>` itself, not internally...
eg.: `<SfModal v-model="isModalVisible"/>`